scheduler/bin/help-delete-temporary-clusters

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Usage: ./bin/make-gke-test-cluster <project> <zone> <clustername>
+#   Configure a kubernetes cluster for running pool-based integration tests and running pools in general.
+#    NOTE: This script labels any clusters it creates and will DELETE old clusters it created.
+#   <project> is a gcloud project.
+#   <zone> can be a zone. E.g., us-central1-a
+#   <clustername> is the name of a cluster. E.g., 'test-cluster-1'
```

Comment:
This docstring doesn't seem to match the script

Reply:
fixed.